README.adoc

```diff
@@ -148,14 +147,14 @@ You can now install the dependencies:
 
 === Convert AsciiDoc into slides
 
-Once the dependencies are installed, verify that the `asciidoctor` command is available.
+Once the dependencies are installed, verify that the `asciidoctor-revealjs` command is available.
 On Linux and macOS, open a terminal and type:
 
- $ $(npm bin)/asciidoctor --version
+ $ npx asciidoctor-revealjs --version
```

Comment:
I saw that npx was core node now. Is it ok to use it? Is it in widely used distros now?

Reply:
> I saw that npx was core node now. Is it ok to use it?

Yes :+1: 

> Is it in widely used distros now?

Not sure, it's included in `npm` so I guess it should be available when you install node/npm from your package manager.